package.json

```diff
@@ -8,13 +8,13 @@
   },
   "scripts": {
     "preinstall": "npm update eos-icons ",
-    "postinstall": "grunt -b ./src moveFiles && cp node_modules/eos-icons/animated-svg/*.svg ./src/svg/",
+    "postinstall": "grunt -b ./src moveFiles",
     "build": "tsc --project ./",
-    "start:dev": "grunt -b ./src moveFiles && NODE_PATH=./src nodemon ./src/server.ts",
-    "start": "grunt -b ./src moveFiles && NODE_PATH=./src ts-node --transpile-only ./src/server.ts",
+    "start:dev": "NODE_PATH=./src nodemon ./src/server.ts",
+    "start": "NODE_PATH=./src ts-node --transpile-only ./src/server.ts",
     "lint": "eslint \"src/*\"",
     "lint:fix": "eslint \"src/*\" --fix",
-    "test:v1": "grunt -b ./src moveFiles && NODE_ENV=test NODE_PATH=./src ts-node --transpile-only ./src/server.ts"
```

Comment:
I've removed the `moveFiles` call from the scripts, since it will be called from inside the update process.

Reply:
I've kept the moveFiles call only for the `test` environment, since the update process will not be fired in the test environments, now all tests are passing.